test/setup.ts

```diff
@@ -25,4 +25,5 @@ export default () => {
   //  feature flags
   process.env.SHOW_SERVICE_OFFLINE_PAGE = "false";
   process.env.FEATURE_FLAG_ENABLE_SAVE_AND_RESUME_17102022 = "true";
+  process.env.FEATURE_FLAG_ENABLE_UPDATE_STATEMENT_VALIDATION_05072023 = "true";
```

Comment:
We should default to false. Danger of turning this on somewhere by default.

Reply:
I'll just remove the flag from here altogether and mock isActiveFeature 👍